src/lib.rs

```diff
@@ -187,6 +192,36 @@ impl Device {
         }
     }
 
+    pub fn usb_purge_tx_buffer(&mut self) -> Result<()> {
+        let result = unsafe { ffi::ftdi_usb_purge_tx_buffer(self.context) };
+        match result {
+            0 => Ok(()),
+            -1 /* read */ | -2 /* write */ => Err(Error::RequestFailed),
+            -3 => unreachable!("uninitialized context"),
+            _ => Err(Error::unknown(self.context)),
+        }
+    }
+
+    pub fn usb_set_event_char(&mut self, value: u8, enable: bool) -> Result<()> {
+        let result = unsafe { ffi::ftdi_set_event_char(self.context, value, enable.into()) };
+        match result {
+            0 => Ok(()),
+            -1 /* read */ | -2 /* write */ => Err(Error::RequestFailed),
+            -3 => unreachable!("uninitialized context"),
+            _ => Err(Error::unknown(self.context)),
+        }
+    }
+
+    pub fn usb_set_error_char(&mut self, value: u8, enable: bool) -> Result<()> {
```

Comment:
Does `enable` mean that the value does not matter if `enable` is false?
If so, please make it `pub fn set_error_char(value: Option<u8>) -> Result<()>`

Reply:
ditto...

Reply:
The same as for event characters, let's go with Option for now.

Reply:
Done